projects/js-packages/connection/components/connection-error-notice/index.jsx

```diff
@@ -20,15 +19,39 @@ const ConnectionErrorNotice = props => {
 		restoreConnectionError,
 	} = props;
 
+	const icon = (
+		<Icon
+			icon={
+				<svg
```

Comment:
Here we normally use the `SVG` and `Path` components exported from `@wordpress/components`:

Example of usage: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/blocks/tiled-gallery/icons.js

Reply:
Addressed in 1874cda593a7587c7273bfa538554cf74dc7c461